cactusbot/handler.py

```diff
@@ -78,8 +92,11 @@ def translate(self, packet, handler):
 
         Parameters
         ----------
-        packet : :obj:`Packet`, :obj:`str`, :obj:`tuple`, :obj:`list`,
-                 :obj:`StopIteration` or :obj:`None`
+        packet : :obj:`Packet` is immediately yielded,
+                 :obj:`str` is converted into a text field in a `MessagePacket`
+                 :obj:`tuple` / :obj:`list` is iterated over, yields each item
+                 :obj:`StopIteration` stops future packets from being passed or
+                 :obj:`None` is an ignored packet
```

Comment:
The `Parameters` section was fine as it was before. Put this documentation between `Parameters` and `Examples` block. ([Here's the example for NumPy doc formatting.](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html))

![image](https://cloud.githubusercontent.com/assets/12002323/23576022/1204e77a-0067-11e7-996f-baf940241e8d.png)